src/test/java/uk/gov/justice/digital/domains/DomainRepositoryTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.digital.domains;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.spark.SparkConf;
+import org.apache.spark.sql.SparkSession;
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.digital.config.BaseSparkTest;
+import uk.gov.justice.digital.repository.DomainRepository;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class DomainRepositoryTest {
+
+    protected static SparkSession getConfiguredSparkSession(SparkConf sparkConf) {
```

Comment:
This should extend the base spark test class instead of creating its own spark session.